vhdbuilder/packer/install-dependencies.sh

```diff
@@ -320,7 +320,7 @@ installAndConfigureArtifactStreaming() {
   # arguments: package name, package extension
   PACKAGE_NAME=$1
   PACKAGE_EXTENSION=$2
-  MIRROR_PROXY_VERSION='0.2.9'
+  MIRROR_PROXY_VERSION='0.2.10'
```

Comment:
can/should we maintain this via components.json?

Reply:
yes, that's my plan